diff --git a/lib/url-state-machine.js b/lib/url-state-machine.js
index c25dbc2c486289fbd7446baed24dc6343f0226f6..526db3e7e1c1dea1b221ae4548ed6ea2d3da4740 100644
--- a/lib/url-state-machine.js
+++ b/lib/url-state-machine.js
@@ -1,5 +1,5 @@
 "use strict";
-const punycode = require("punycode");
+const punycode = require("../../punycode");
 const tr46 = require("tr46");
 
 const specialSchemes = {
